index.html

```diff
@@ -14,119 +14,157 @@
+          /></span>
           <a
-            href="https://twitter.com/realdevsquad"
+            href="https://twitter.com/realdevsquad?lang=en"
             target="_blank"
-            rel="nonreferrer"
+            class="social-handles"
+            id="twitter"
             >Twitter</a
-          >&nbsp;
-          <span class="link-seperators">&vert;</span>
-
-          &nbsp;
+          >
+          <span
+            ><img
+              src="../img\re-designed images\twitter logo.png"
+              alt="Twitter logo"
+              class="handles-logos"
+          /></span>
+          <span
+            ><img
+              src="../img\re-designed images\separator icon.png"
+              alt="separator icon"
+              class="separator"
+          /></span>
           <a
-            href="https://www.facebook.com/Real-Dev-Squad-108713777585062"
+            href="https://www.facebook.com/Real-Dev-Squad-108713777585062/"
             target="_blank"
-            rel="nonreferrer"
+            class="social-handles"
+            id="facebook"
+            >Facebook</a
           >
-            Facebook
-          </a>
-        </p>
-        <h3 class="info-note">
-          To know how this page is getting developed, join our Discord Channel.
-          Contact one of our existing
-          <a href="https://members.realdevsquad.com/">members</a> for the
-          invitation link.
-        </h3>
+          <span
+            ><img
+              src="../img\re-designed images\facebook logo.png"
+              alt="Facebook logo"
+              class="handles-logos"
+          /></span>
+          <span
+            ><img
+              src="../img\re-designed images\separator icon.png"
+              alt="separator icon"
+              class="separator"
+          /></span>
+          <a
+            href="https://www.instagram.com/realdevsquad/?hl=en"
+            target="_blank"
+            class="social-handles"
+            id="instagram"
+            >Instagram</a
+          >
+          <span
+            ><img
+              src="../img\re-designed images\instagram logo.png"
+              alt="Instagram logo"
+              class="handles-logos"
+          /></span>
+        </div>
       </div>
-    </main>
-
-    <footer>
-      <p class="info-repo">
-        The contents of this website are deployed from this
-        <a
-          href="https://github.com/Real-Dev-Squad/website-welcome"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          open sourced repo
-        </a>
-      </p>
-    </footer>
 
+      <img
+        class="welcome-image"
+        src="../img\re-designed images\welcome-image.png"
+        alt="welcome-image"
+      />
+      <div class="news-feeds-container">
+        <div class="new-tag">New</div>
+        1 Year Aniversary--New Features Available for YOU!
+        <a id="checkout-tag" href="https://www.google.com">Check them out </a>
```

Comment:
Can we have this link replaced by this? `https://www.realdevsquad.com/releases`

Reply:
> Can we have this link replaced by this? `https://www.realdevsquad.com/releases`

Sure!

Reply:
> Can we have this link replaced by this? `https://www.realdevsquad.com/releases`

sure!